src/style.css

```diff
@@ -0,0 +1,120 @@
+/* stylelint-disable declaration-block-trailing-semicolon */
+/* stylelint-disable shorthand-property-no-redundant-values */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+  --background-color: #fff;
+  --border-color: #f3f3f3;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--border-color);
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__goods-img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px
+}
+
+.product-card__discription {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.product-card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+}
+
+.product-card__rate {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  align-items: center;
+}
+
+.product-card__star-container {
+  display: flex;
+}
+
+.product-card__star {
+  background: url(./images/star-active.svg) no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.product-card__star:nth-last-child(5n-4) {
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.product-card__review {
+  font-weight: 400;
+  text-align: right;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-text-color);
+  margin: 0;
+}
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.card-price__word {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card-price__cost {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.product-card__button {
+  font-weight: 700;
+  font-size: 14px;
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  justify-content: center;
+  color: var(--background-color);
+  background: var(--button-color);
+  border-radius: 5px;
+  margin-top: 16px;
+  width: 166px;
+  height: 40px;
+  transition: 0.7s;
+}
+
+.product-card__button:hover {
+  border: 1px solid var(--button-color);
```

Comment:
why do we need to change border on hover? add this style directly to your button